modules/deploy/pages/deployment-option/self-hosted/kubernetes/k-deployment-overview.adoc

```diff
@@ -5,106 +5,118 @@
 
-- The StatefulSet to manage the Redpanda brokers.
-- A Headless ClusterIP Service for internal communication with the Redpanda cluster.
-- A NodePort Service for external communication with the Redpanda cluster.
+- A StatefulSet to manage Redpanda brokers
+- A Headless ClusterIP Service for internal communication with the Redpanda cluster
+- A NodePort Service for external communication with the Redpanda cluster
 
-The Redpanda Helm chart comes with default settings that enable a straightforward deployment out of the box. However, Helm also offers customization by allowing you to override these default values.
 
-You can override the defaults in your own YAML files using the `--values` option or directly in the command line with the `--set` option. These changes in Helm chart values not only influence the configuration of the Redpanda cluster, but also determine what Kubernetes resources are deployed and how these resources are configured.
+[[helm-and-redpanda-operator]]
+=== Redpanda Operator
 
-=== Helm and Redpanda Operator
+The Redpanda Operator provides two deployment modes controlled by the `useFlux` flag. The modes differ in how reconciliation is handled:
 
-The recommended option for deploying Redpanda in Kubernetes is a combination of <<Helm>> and the Redpanda Operator. While the Helm chart facilitates the deployment of Redpanda, the Redpanda Operator manages the lifecycle of Redpanda clusters. The operator uses https://fluxcd.io/flux/concepts/[Flux^] to automate the deployment of the Redpanda Helm chart through the HelmRelease resource.
+- *Flux-managed mode (`useFlux: true`)*:
+When `useFlux` is set to `true` (default), the Redpanda Operator delegates reconciliation to https://fluxcd.io/flux/concepts/[Flux^] controllers. The workflow is as follows:
++
+. The Redpanda Operator defines the necessary CRDs for Redpanda. A Redpanda resource represents a Redpanda cluster, configured through the Redpanda Helm chart values.
+. When the Redpanda Operator detects changes to Redpanda resources, it triggers Flux to reconcile the corresponding HelmRelease resource.
+. Flux uses the HelmRelease resource to manage the lifecycle of the Redpanda Helm chart, applying updates as needed.
 
-Flux, when integrated with the Redpanda Operator, uses the HelmRelease resource to manage the lifecycle of the Redpanda Helm chart in the Kubernetes cluster. This integration streamlines the creation and management of Redpanda clusters, ensuring they are always in the desired state as defined by the operator's custom resource definitions (CRDs).
+- *Redpanda-managed mode (`useFlux: false`)*:
+When `useFlux` is set to `false`, the Redpanda Operator manages and reconciles Redpanda resources directly, without involving Flux. This mode works as follows:
++
+. The Redpanda Operator creates both `HelmRelease` and `HelmRepository` resources but suspends them, keeping Flux inactive.
```

Comment:
It might be worth calling out that despite the existence of this flag, the operator still requires the Flux CRDs to be installed. Phrased in a way that indicates we intend to change that soon :)